### PR TITLE
Dockerfile: make env into a single layer

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,9 @@ ADD package.json package-lock.json /tmp/cirrus-ci-web/
 
 RUN npm ci
 
-ENV GENERATE_SOURCEMAP true
-ENV NODE_ENV production
+ENV \
```
Comment (fkorotkov) [inline Dockerfile]: Why extra `\`?

Comment (RDIL) [reply Dockerfile]: Because it makes it look more even

Comment (fkorotkov) [reply Dockerfile]: It seems you'll still have the same indention. According to documentation you probably also want to use `=`: https://docs.docker.com/engine/reference/builder/#env

Plus in the example in the docs they even user single line syntax:

```Dockerfile
ENV NODE_ENV=production GENERATE_SOURCEMAP=true
```

**Comment by fkorotkov** [comment]
Thanks!

**Pull request merged**